demoRefixation.py

```diff
@@ -0,0 +1,55 @@
+from EMRM import *
+import random
+import string
```

Comment:
doesn't really matter, but it's standard to import standard modules like these two first, before custom ones like EMRM